RIOT-OS/RIOT
# boards/nucleo-l4r5zi and l496zg: add pinout to documentation page

### Contribution description

This PR adds pinout diagrams from MB1312 for RIOT supported boards:
- nucleo-l4r5zi,
- nucleo-l496zg.

### Testing procedure

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-l4r5zi.html
xdg-open doc/doxygen/html/group__boards__nucleo-l496zg.html
```

### Issues/PRs references

None
